### ISSUE TITLE
Trailing spaces/tabs on entering closing brace

### ISSUE THREAD

**Issue by rchl** [colinta/SublimeBracketeer#6]
Steps:

1. Caret at `|`
```js
state('redirect-store2', {
    url: '/{countryCode:[a-z]{2}}/store{path:.*}',
    controller: storeRedirectController,|
```

2. Press `enter` key:
```js
state('redirect-store2', {
    url: '/{countryCode:[a-z]{2}}/store{path:.*}',
    controller: storeRedirectController,
    |
```

3. Type `}`:
```js
state('redirect-store2', {
    url: '/{countryCode:[a-z]{2}}/store{path:.*}',
    controller: storeRedirectController,
}|<trailing space>
```

Expected: No trailing space.
Actual: One space is left after the closing brace.

(It happens either with using spaces or tabs for indentation)

**Comment by colinta** [comment]
This is a really great bug report!  I somehow missed the notification email when you opened the issue, but I'm investigating now.

**Comment by colinta** [comment]
OK this should be fixed in 2.1.2 – thanks again for a good, easily reproducible example case.
